eoscostarica/lifebank
# Fix/general bug

Resolves #647 
Resolves #648  
Resolves #649  

### What does this PR do?
- Fix sidebar is ghost query creation that slowdown the page speed
- Sponsor name is not been updated when signing up a new one
- Email verified validation was removed in later changes, get it back

### How should this be tested?
- Make run
- Go to sidebar and check that there no console error query warning
- Signup as sponsor and check that start name is equal after accepting the consent contract in the profile view
- Check can't sign in after a sign up without hadn't had verified the email
